### PR TITLE
making layout work in older browsers

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

File: header/_header-ie8.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.header--new {
+
+	.o-header__masthead a {
+		height: 46px;
```
Comment (i-like-robots) [inline header/_header-ie8.scss]: worth a comment explaining magic number?

File: header/_header-ie8.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	}
+
+	.o-header__left-tools__link-search {
+		display: block;
+	}
+
+	.o-header__left-tools__link {
+		font-size: 0.8em;
```
Comment (i-like-robots) [inline header/_header-ie8.scss]: font size mixin?

File: header/_header-new.scss [hunk 3]
```diff
@@ -111,10 +118,12 @@
 
 	.o-header__left-tools__link-search:before {
 		@include oIconsGetIcon('search', oColorsGetPaletteColor($icon_color), $icon_size);
+		// the above mixin sets display to inline-block, hence...
 		display: block;
 
 		@include oGridRespondTo(L) {
 			@include oIconsGetIcon('search', oColorsGetPaletteColor($icon_color), $icon_size_l);
+			// the above mixin sets display to inline-block, hence...
```
Comment (i-like-robots) [inline header/_header-new.scss]: =]

Comment (i-like-robots) [inline header/_header-new.scss]: Add the argument `$apply-base-styles: false` to the mixin here

File: header/_header-ie8.scss (hunk 2 above)
Comment (aintgoin2goa) [inline header/_header-ie8.scss]: is there one?  what is it?

Comment (i-like-robots) [inline header/_header-ie8.scss]: `oTypographySans('xs')` maybe?

**Pull request merged**